autogpt_platform/frontend/src/hooks/useAgentGraph.tsx

```diff
@@ -159,96 +150,93 @@ export default function useAgentGraph(
-              blockCosts: block.costs,
-              categories: block.categories,
-              description: block.description,
-              title: `${block.name} ${node.id}`,
-              inputSchema: block.inputSchema,
-              outputSchema: block.outputSchema,
-              hardcodedValues: node.input_default,
-              webhook: node.webhook,
-              uiType: block.uiType,
-              connections: graph.links
-                .filter((l) => [l.source_id, l.sink_id].includes(node.id))
-                .map((link) => ({
-                  edge_id: formatEdgeID(link),
-                  source: link.source_id,
-                  sourceHandle: link.source_name,
-                  target: link.sink_id,
-                  targetHandle: link.sink_name,
-                })),
-              backend_id: node.id,
+              edgeColor: getTypeColor(
+                getOutputType(newNodes, link.source_id, adjustedSourceName!),
+              ),
+              sourcePos: newNodes.find((node) => node.id === link.source_id)
+                ?.position,
+              isStatic: link.is_static,
+              beadUp: 0,
+              beadDown: 0,
             },
+            markerEnd: {
+              type: MarkerType.ArrowClosed,
+              strokeWidth: 2,
+              color: getTypeColor(
+                getOutputType(newNodes, link.source_id, adjustedSourceName!),
+              ),
+            },
+            source: link.source_id,
+            target: link.sink_id,
+            sourceHandle: adjustedSourceName || undefined,
+            targetHandle: link.sink_name || undefined,
           };
-          return newNode;
-        });
-        const newNodes = _newNodes.filter((n) => n !== null);
-        setXYEdges(() =>
-          graph.links.map((link) => {
-            const adjustedSourceName = cleanupSourceName(link.source_name);
-            return {
-              id: formatEdgeID(link),
-              type: "custom",
-              data: {
-                edgeColor: getTypeColor(
-                  getOutputType(newNodes, link.source_id, adjustedSourceName!),
-                ),
-                sourcePos: newNodes.find((node) => node.id === link.source_id)
-                  ?.position,
-                isStatic: link.is_static,
-                beadUp: 0,
-                beadDown: 0,
-              },
-              markerEnd: {
-                type: MarkerType.ArrowClosed,
-                strokeWidth: 2,
-                color: getTypeColor(
-                  getOutputType(newNodes, link.source_id, adjustedSourceName!),
-                ),
-              },
-              source: link.source_id,
-              target: link.sink_id,
-              sourceHandle: adjustedSourceName || undefined,
-              targetHandle: link.sink_name || undefined,
-            };
-          }),
-        );
-        return newNodes;
-      });
-    },
-    [availableBlocks, availableFlows, getOutputType],
-  );
```

Comment:
No changes inside this function, I just renamed it to make sure that we don't have to include it as a dependency in hooks where it's used, and so it's clear that this is not exported.